integr8ly/workload-web-app
# Increase replicas, add podDisruptionBugdet and topologySpreadConstraints

## What
increase replicas, add podDisruptionBugdet and topologySpreadConstraints
## Verification
- deploy managed api on a multi az cluster
- checkout this branch 
- set the env
```bash   
export RHOAM=true
export GRAFANA_DASHBOARD=true

```
- run `make local\deploy`
- check that there are two pods for the workload-web-app
- Check that the pods are on different AZ using integreatly-operator/scripts/podsAz.sh  e.g.                                                                                        
```bash
./podsAz.sh workload-web-app           
Pods distribution for 'workload-web-app'
| Pod name | Availability Zone |
| -------- | ----------------- |
| workload-web-app-1-7hwrm | eu-west-1a |
| workload-web-app-1-deploy | eu-west-1c |
| workload-web-app-1-r9bcc | eu-west-1c |
```
- check that there is a podDisruptionBugdet
```bash
oc get podDistruptionBugdet --namespace workload-web-app
```
- run `make local\undeploy` to uninstall and confirm removed
